### ISSUE TITLE
[TIL]이윤회_201223

### ISSUE THREAD

**Issue by yoonhoelee** [codestates/ds-TIL#1243]
- **키워드**: Deep Learning, weight decay, learning rate

- **배운 것**: Multi-layer network에서 하이퍼 파라미터를 튜닝해 모델의 성능을 높이는 다양한 방법들에 대해서 학습했다.

- **어려웠던 부분**: 이미지 파일을 처음 다뤄보는 것인데, 이미지 파일의 차원에 대한 개념이 조금은 어려웠다. 또한 임의의 이미지 파일에서 숫자를 인식하는 것에 해당하는 코드를 이해하는 것이 어려웠다.

- **더 알고 싶은 부분 / 공부하고 싶은 부분**: mnist를 활용해 이미지를 예측하는 코드 부분을 아직은 이해하지 못했다. 모델을 만드는 부분 까지는 알겠으나, 그 뒤 내 파일을 넣는 부분에서는 조금 해맸던것 같다. 이 코드에 대해 많은 공부가 필요할 것 같다.

- **느낀 점**: 오늘 역시 내용이 쉽지는 않았던 것 같다. 비록 정확도는 형편 없었지만, 내가 임의로 가져온 이미지를 인식하는 것이 신기했다. 공부를 하면서 재밌다는 감정을 느낀 것은 오랜만인것 같다. 내일 배울 내용이 기대된다.
